src/components/Announcements/AnnouncementModal.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <transition name="modal-fade">
+    <div v-show="open" class="modal-backdrop">
+      <div class="modal">
+        <div class="announce-header">
+          <div class="announce-title">{{announcement.title}}</div>
+          <div class="announce-date">{{ toStringDate(announcement.created) }}</div>
+        </div>
+        <div class="announce-body">{{announcement.description}}</div>
+        <div
+            class="close-btn"
+            @click="close"
+            >
+          X
+        </div>
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import DateUtils from '../../utils/DateUtils';
+
+export default {
+  name: 'AnnouncementModal',
+  props: {
+    open: {
+      type: Boolean,
+      required: true,
+    },
+    announcement: {
+      type: Object,
+    },
+  },
+  methods: {
+    close() {
+      this.$emit('on-close-announcement');
+    },
+    toStringDate(date) {
+      return DateUtils.toStringDate(date);
+    },
+  },
+};
+</script>
+
+<style scoped>
+  .modal-fade-enter,
+  .modal-fade-leave-active {
+    opacity: 0;
+  }
+  .modal-fade-enter-active,
+  .modal-fade-leave-active {
+    transition: opacity .5s ease
+  }
+
+  .modal-backdrop {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background-color: rgba(0, 0, 0, 0.3);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .modal {
+    position: fixed;
+    background: white;
+    border: solid 12px rgba(248, 134, 52, 0.5);
+    width: 500px;
+    max-height: 500px;
+
+    overflow-x: visible;
+    display: flex;
+    flex-direction: column;
+    padding: 16px;
+  }
+
+  .announce-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+  }
+  .announce-title {
+    flex-grow: 3;
+    font-weight: bold;
+    font-size: 1.1rem;
+    text-align: left;
+  }
+  .announce-date {
+    text-align: right;
+  }
+  .announce-body {
+    margin-top: 16px;
+    text-align: left;
+  }
+
+  .close-btn {
+    background: rgb(252,195,154);
+    cursor: pointer;
+    position: absolute;
+    right: -20px;
```

Comment:
negative pixels!!!!!!!!!!!!!!!

Reply:
Yeah just so much easier than trying to get that corner circle thing wrapped in a larger container